src/containers/Tenant/Diagnostics/Overview/Overview.tsx

```diff
@@ -92,11 +92,14 @@ function Overview({type, path, database}: OverviewProps) {
         return <Loader size="m" />;
     }
 
-    if (schemaError || overviewError) {
-        return <ResponseError error={schemaError || overviewError} />;
-    }
-
-    return renderContent();
+    return (
+        <React.Fragment>
+            {schemaError || overviewError ? (
+                <ResponseError error={schemaError || overviewError} />
+            ) : null}
+            {renderContent()}
```

Comment:
There are two errors now - one from error here, and one from component, that displays content

<img width="235" alt="Screenshot 2024-07-30 at 11 13 29" src="https://github.com/user-attachments/assets/0894d779-9611-4ff5-a176-18801ad1ca68">

Reply:
Yes, there can be two messages here. One request may fail with an error (or we interpret empty data as an error), while another may fail with no error but no data.